examples/website/openstreet-map/app.js

```diff
@@ -0,0 +1,70 @@
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+
+import DeckGL, {TileLayer, BitmapLayer} from 'deck.gl';
+
+export const INITIAL_VIEW_STATE = {
+  latitude: 47.65,
+  longitude: 7,
+  zoom: 4.5,
+  maxZoom: 16,
+  pitch: 50,
+  bearing: 0
+};
+
+const s = 'c';
+
+// https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames#Lon..2Flat._to_tile_numbers_2
+function tile2latLng(x, y, z) {
+  const lng = (x / Math.pow(2, z)) * 360 - 180;
+  const n = Math.PI - (2 * Math.PI * y) / Math.pow(2, z);
+  const lat = (180 / Math.PI) * Math.atan(0.5 * (Math.exp(n) - Math.exp(-n)));
+  return [lng, lat];
+}
+
+function tileIndexToBounds({x, y, z}) {
+  const [west, north] = tile2latLng(x, y, z);
+  const [east, south] = tile2latLng(x + 1, y + 1, z);
+
+  return [west, south, east, north];
+}
+
+export class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  _renderLayers() {
+    return [
+      new TileLayer({
+        renderSubLayers: props => {
+          const {x, y, z} = props.tile;
+
+          return new BitmapLayer({
+            id: `bitmap-${props.id}`,
+            image: `https://${s}.tile.openstreetmap.org/${z}/${x}/${y}.png`,
+            bounds: tileIndexToBounds({x, y, z})
```

Comment:
Just a thought: the lnglat bounds of a tile could be something we provide as part of the `Tile` class. I can see a lot of use cases needing these numbers.